src/explore-education-statistics-admin/src/components/editable/EditableContentBlock.tsx

```diff
@@ -150,6 +192,7 @@ const EditableContentBlock = ({
 
   return (
     <>
+      {showAltTextMessage && <AltTextWarningMessage />}
```

Comment:
Relatively minor, but would just have this next to the `EditableBlockLockedMessage` to make it more obvious